src/test/.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
```

Comment:
Currently this won't work, right? Since we don't have steps yet to setup the app itself locally on CI?